libs/design-system-components/src/components/Editor/formatting-buttons.tsx

```diff
@@ -44,7 +44,7 @@ export const MarkButton: React.FC<ToolbarButtonProps> = ({ format, icon, style }
   const active = CustomEditor.isMarkActive(editor, format);
   return (
     <button
-      onClick={event => {
+      onMouseDown={event => {
```

Comment:
this event does not trigger on mobile devices

Reply:
Thats right

Reply:
good catch, I changed it to test something and forgot to revert